### DISCUSSION TITLE
The way to show dates and times on the History pane

### DISCUSSION THREAD

**Discussion by kazikikaziki** [community/community#24071] [Apps, API and Webhooks]
<p>It shows date (without time) or how many days (hours,miutes) ago.<br>
But i’m always interested in DATE and TIME!</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/a/c/aca33c69ee53103a2190667df43d7219ff73730f.png" data-download-href="/uploads/short-url/oDdR63RUP2HwmKhPx8FMW5zhNw3.png?dl=1" title="github-desktop-screenshot-windows2" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/52305677/181081804-9bb20452-aa63-4aa9-ae2c-b7d99f5c6aae.png" alt="github-desktop-screenshot-windows2" data-base62-sha1="oDdR63RUP2HwmKhPx8FMW5zhNw3" width="690" height="474" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/a/c/aca33c69ee53103a2190667df43d7219ff73730f_2_690x474.png, https://aws1.discourse-cdn.com/github/optimized/3X/a/c/aca33c69ee53103a2190667df43d7219ff73730f_2_1035x711.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/a/c/aca33c69ee53103a2190667df43d7219ff73730f_2_1380x948.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/a/c/aca33c69ee53103a2190667df43d7219ff73730f_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">github-desktop-screenshot-windows2</span><span class="informations">1920×1320 377 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by steveward** [comment, marked as answer]
<p><a class="mention" href="/kazikikaziki">@kazikikaziki</a> thanks for the question! If you hover over the date you will get the full timestamp showing the commit time. I hope that helps!</p>

**Comment by kazikikaziki** [comment]
<p>i got it. thank you!!</p>
